ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/GameplayProgrammer.asset

```diff
@@ -0,0 +1,36 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: d8cfffae36be18b40b59e8e84d47a6ad, type: 3}
+  m_Name: GameplayProgrammer
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: 767ac96fd5784e2438df8b3aca9a28f3, type: 2}
+  positionName: Gameplay Programmer
+  jobSummaryLines:
+  - "J\u016Bkend\u014D Co., Ltd. is now hiring for a"
```

Comment:
Same here, I'd do this in general. That way companies and positions can be developed separately.